docs/hub/security.md

```diff
@@ -4,6 +4,54 @@ title: Security and trust on the Hugging Face Hub
 
 # Security and trust on the Hugging Face Hub
 
+## User Access Tokens
+
+### What are User Access Tokens?
+
+User Access Tokens are the preferred way to authenticate an application or a notebook to the HuggingFace services. You can manage your access tokens in your [settings](https://huggingface.co/settings/tokens).
+
+![/docs/assets/hub/access-tokens.png](/docs/assets/hub/access-tokens.png)
+
+Access tokens allow applications and notebooks to impersonate you, with restricted permissions. Each token is given a role, among the following:
+
+- `read`: tokens with this role can only be used to provide read access to repositories you could read. That includes public repositories and private repositories that you or an org you're a member of own. Use this role if you only need to read content from the HuggingFace hub (eg, when doing inference).
+
+- `write`: tokens with this role additionally grant write access to the repositories you have write access to. Use this token if you need to create or push content to a repository (eg, when training a model).
+
+### How to manage User Access Tokens?
+
+To create an access token, go to your settings, then click on the ["Access Tokens" tab](https://huggingface.co/settings/token). Click on the "New token" button to create a new User Access Token.
+
+![/docs/assets/hub/new-token.png](/docs/assets/hub/new-token.png)
+
+Select a role and a name for your token and voilà - you're ready to go!
+
+You can delete an refresh User Access Tokens by clicking on the "Manage" button.
+
+![/docs/assets/hub/delete-token.png](/docs/assets/hub/delete-token.png)
+
+### How to use User Access Tokens?
+
+There are plenty of ways to use a User Access Token to access the Hugging Face hub, granting you the flexibility you need to build awesome apps on top of it.
+
+- User Access Tokens can be used **in place of a password** to access the Hugging Face hub with git or with Basic authentication
+- User Access Tokens can be passed as a **bearer token** when calling the Hugging Face APIs
+- User Access Tokens can be used in the Hugging Face Python library:
```

Comment:
Not 100% sure we should link to that page

```suggestion
- User Access Tokens can be used in the `huggingface_hub` [Python library](https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub):
```

Reply:
Shoot I wanted to say "libraries", to refer to `datasets`, `transformers` and `huggingface_hub` all at once